flink-tests/src/test/java/org/apache/flink/test/state/operator/restore/AbstractOperatorRestoreTestBase.java

```diff
@@ -104,16 +123,32 @@ public abstract class AbstractOperatorRestoreTestBase extends TestLogger {
                             .setNumberSlotsPerTaskManager(NUM_SLOTS_PER_TM)
                             .build());
 
-    private final boolean allowNonRestoredState;
```

Comment:
I checked why we don't need that anymore. Apparently, this feature was added with FLINK-7595 and removed for Flink 1.4 with FLINK-8472. I'm gonna leave it like that. I guess it's fine to remove it :+1: